programs/server/Server.cpp

```diff
@@ -1488,6 +1488,7 @@ int Server::main(const std::vector<std::string> & /*args*/)
         /// We load temporary database first, because projections need it.
         database_catalog.initializeAndLoadTemporaryDatabase();
         loadMetadataSystem(global_context);
+        maybeConvertOrdinaryDatabaseToAtomic(global_context, DatabaseCatalog::instance().getSystemDatabase());
```

Comment:
maybe make this conversion optional?

Reply:
It is optional: 
https://github.com/ClickHouse/ClickHouse/blob/94dd80e90069bca17066abadb3bf60d3ff496e59/src/Interpreters/loadMetadata.cpp#L282-L283
(I don't want to introduce another setting for it)